syntax/rmd.vim

```diff
@@ -113,10 +113,9 @@ syn region rmdCodeBlock matchgroup=rmdCodeDelim start="^\s*```.+$" matchgroup=rm
 hi link rmdCodeBlock Special
 
 " Now highlight chunks:
-syn region knitrOption start='^#| ' end='$' contained  containedin=rComment,pythonComment contains=knitrVar,knitrValue transparent
-syn match knitrValue ': \zs.*\ze$' keepend contained containedin=knitrOption
-syn match knitrVar '| \zs\S\{-}\ze:' contained containedin=knitrOption
```

Comment:
In order to make the code more understandable, I renamed these items in a way that it is clear that knitr options in the body of the chunk are addressed, but not knitr options in the header of the chunk.